data/json/items/ranged/slings.json

```diff
@@ -6,7 +6,7 @@
     "subtypes": [ "GUN" ],
     "symbol": "(",
     "color": "brown",
-    "name": { "ctxt": "weapon", "str": "sling" },
+    "name": { "ctxt": "weapon", "str": "shepherd's sling" },
```

Comment:
We discussed this,  I don't agree that this clarified anything,  sling is fine,  or short sling to distinguish it from the long sling.